inference-engine/src/low_precision_transformations/src/network_helper.cpp

```diff
@@ -800,6 +800,23 @@ FakeQuantizeDequantization NetworkHelper::getDequantization(const std::shared_pt
     return FakeQuantizeDequantization(dataNode, convert, subtract, multiply);
 }
 
+void NetworkHelper::normalizeDequantization(FakeQuantizeDequantization& dequantization) {
```

Comment:
I don't think changing value via reference is clear here. Can you please explore: is it typical way for nGraph/LPT library functions to update value by reference? If not, please, make `dequantization` constant and return new `FakeQuantizeDequantization` instance.